zombies.js

```diff
@@ -1,3 +1,162 @@
+// jshint esversion: 6
+class Item {
+  constructor(name) {
+    this.name = name;
+  }
+}
+
+class Weapon extends Item {
+  constructor(name, damage) {
+    super(name);
+    this.damage = damage;
+  }
+}
+
+class Food extends Item {  
+  constructor(name, energy) {
+    super(name);
+    this.energy = energy;
+  }
+}
+
+class Player {
+  constructor(name, health, strength, speed) {
+    this._pack = [];
+    this._maxHealth = health;
+    this.name = name;
+    this.health = health;
+    this.strength = strength;
+    this.speed = speed;
+    this.isAlive = true;
+    this.equipped = false;
+  }
+
+  getPack() {
+    return this._pack;
+  }
+
+  getMaxHealth() {
+    return this._maxHealth;
+  }
+
+  checkPack() {
+    let playerPack = this._pack;
+    console.log('Contents: ');
+    for (let i in playerPack.length) {
+      console.log(playerPack[i] + ", ");
+    }
+  }
+
+  takeItem(item) {
+    let packContents = this._pack;
+    if (packContents.length > 2 && !packContents.includes(item)) {
```

Comment:
There isnt a specific requirement that would require 
```
&& !packContents.includes(item)
```
Did you come across an issue by not having this check?